utils/split_sentence.py

```diff
@@ -30,7 +30,7 @@ def get_sheet():
             creds = pickle.load(token)
     # If there are no (valid) credentials available, let the user log in.
     if not creds or not creds.valid:
-        if creds and creds.expired and creds.refresh_token:
+        if creds.expired and creds.refresh_token:
```

Comment:
Not sure how python works, but what I concern is that we have already check `if not creds` condition, so we might not need to have condition `if creds`

Reply:
OK, I see, I missed `OR` in the condition 😞, thanks!